pkg/tsdb/cloudwatch/live.go

```diff
@@ -150,6 +150,11 @@ func (e *cloudWatchExecutor) sendLiveQueriesToChannel(req *backend.QueryDataRequ
 
 	if err := eg.Wait(); err != nil {
 		plog.Error(err.Error())
+		responseChannel <- &backend.QueryDataResponse{
+			Responses: backend.Responses{
+				"A": {Error: err},
```

Comment:
The refId may not be `A`, I think that a better solution can be to send errors in `startLiveQuery` that has that information available. In any case, living this to someone with more experience in the area. It'd be great if some user from #35448 can verify if this solves the issue.

Reply:
You are right! I'm moving this down.